app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -121,8 +121,8 @@ def check_enrollment(enrollment)
     enrollment.update(status_check_attempted_at: status_check_attempted_at)
   end
 
-  def passed_with_unsupported_secondary_id_type?(response)
-    return response['secondaryIdType'].present? &&
+  def passed_with_unsupported_secondary_id_type?(enrollment, response)
```

Comment:
I think this method would be easier to read if the conditional were broken up.  Three clauses is a lot reason about.  I also find it harder to reason about a conditional opens with a negation of a boolean.  A refactor could look like one of the below approaches:

1. opening with a guard statement that returns false if the enrollment is enhanced_ipp
2. using an unless statement with enrollment.enhanced_ipp?
3. creating a new helper method that returns true if the enrollment is id_ipp, like `enrollment.id_ipp?`, and replacing `enrollment.enhanced_ipp?` with that helper method

The first option is by far my favorite because it feels the simplest.  Curious what you think, @keith-nava!

Reply:
Nice refactor, thanks @KeithNava.